### PR TITLE
[REEF-571] Remove NuGet for Org.Apache.REEF.Bridge.JAR

### PR DESCRIPTION
**Pull request by anupam128**
This addressed the issue by 
* Removing NuGet package generation from Org.Apache.REEF.Bridge.JAR

JIRA:
  [REEF-571](https://issues.apache.org/jira/browse/REEF-571)

### CODE REVIEW

**Comment by markusweimer** [comment]
I'll do a pass.

File: lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj]: What has changed here?

File: lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj [hunk 2]
```diff
@@ -54,9 +52,7 @@ under the License.
   <!--
     Executes 'mvn install', if the JAR does not exist.
   -->
-  <Target Name="Build" 
-          DependsOnTargets="CheckMavenInstall;CheckJavaInstall;CheckProtocInstall" 
-          Outputs="$(OutputPath)">
+  <Target Name="Build" DependsOnTargets="CheckMavenInstall;CheckJavaInstall;CheckProtocInstall" Outputs="$(OutputPath)">
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj]: This looks like a pure formatting change, please remove.

File: lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj [hunk 3]
```diff
@@ -84,48 +80,10 @@ under the License.
     <Exec Command="$(M2_HOME)\bin\mvn -TC1 -DskipTests clean" Condition="Exists('$(Bridge_JAR)')" WorkingDirectory="$(REEF_Source_Folder)" />
     <Delete Files="$(OutputPath)\$(Bridge_JAR_Name)" />
     <Delete Files="$(OutputPath)\$(Client_JAR_Name)" />
-
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj]: This is a pure formatting change, please remove.

**Comment by markusweimer** [comment]
@anupam128 I've done a pass.

File: lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj (hunk 1 above)
Comment (anupam128) [inline lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj]: I am not sure. Doesn't show any difference when I do a local diff.

**Comment by anupam128** [comment]
Addressed the comments.

**Comment by markusweimer** [comment]
@anupam128 Thanks, I will test and merge

**Pull request closed without merging**